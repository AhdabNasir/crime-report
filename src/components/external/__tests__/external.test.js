import React from 'react';
import renderer from 'react-test-renderer';
import External from '..';

describe('<External/>', () => {
  it('renders <External/> component', () => {
    const tree = renderer
      .create(<External link="www.google.com" name="Google" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
