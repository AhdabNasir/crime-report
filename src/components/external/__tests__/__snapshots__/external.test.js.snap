// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<External/> renders <External/> component 1`] = `
<a
  className="govuk-link"
  href="www.google.com"
  rel="noopener noreferrer"
  target="_blank"
>
  Google
</a>
`;
